config/variables.cfg

```diff
@@ -126,13 +126,36 @@ variable_status_leds_colors: {
             'off': {'r': 0.0, 'g': 0.0, 'b': 0.0, 'w': 0.0},
             'printing': {'r': 1.0, 'g': 0.0, 'b': 0.0, 'w': 0.0},
             'standby': {'r': 0.01, 'g': 0.01, 'b': 0.01, 'w': 0.1},
+            'error': {'r': 0.6, 'g': 0.0, 'b': 0.0, 'w':0.0},
         },
         'nozzle': {
             'heating': {'r': 0.8, 'g': 0.35, 'b': 0.0, 'w':0.0},
             'off': {'r': 0.0, 'g': 0.0, 'b': 0.0, 'w': 0.0},
             'on': {'r': 0.8, 'g': 0.8, 'b': 0.8, 'w':1.0},
             'standby': {'r': 0.6, 'g': 0.0, 'b': 0.0, 'w':0.0},
+            'error': {'r': 0.6, 'g': 0.0, 'b': 0.0, 'w':0.0},
         },
+	# case leds
```

Comment:
Can you remove this comment ?

Reply:
I also change the comment :

# SteathBurner minidisplay and case leds colors (if installed in the machine)
variable_status_leds_colors: {
        'logo': {
            'busy': {'r': 0.4, 'g': 0.0, 'b': 0.0, 'w': 0.0},
            'cleaning': {'r': 0.0, 'g': 0.02, 'b': 0.5, 'w': 0.0},